### PR TITLE
Added get/setParent to ContainerWindow

### PR DESCRIPTION
**Pull request by Sly1024**

### CODE REVIEW

File: packages/desktopjs/src/Default/default.ts [hunk 1]
```diff
@@ -93,6 +93,15 @@ export namespace Default {
             return Promise.reject("Not supported");
         }
 
+        public getParent(): Promise<ContainerWindow> {
+            // on the web, there's no parent
```
Comment (psmulovics) [inline packages/desktopjs/src/Default/default.ts:97]: Is opener a parent?

Comment (Sly1024) [reply packages/desktopjs/src/Default/default.ts:97]: I found [this PR](https://github.com/symphonyoss/ContainerJS/pull/137/commits/707028e712b6016a41c7f69b29176c58f3957ca7#diff-3473b5f3f1ef120a3cd6e2d94c14534eR242) (it's symphony, not openfin - or is it?) where they treat opener as parent.
However, in our case I don't think we should. The parent window:
- is always behind the child windows
- closes its children when closed
- activates/resotres its children when restored

The opener doesn't do any of those.

Comment (bingenito) [reply packages/desktopjs/src/Default/default.ts:97]: Should we blank out the getParent you have in OpenFin here as I think that is the opener?

Comment (Sly1024) [reply packages/desktopjs/src/Default/default.ts:97]: We don't really use getParent, and if they implement setParent correctly, they must "fix" getParent as well. However, to make setParent/getParent calls consistent, I think we should blank it out.

**Pull request merged**